### DISCUSSION TITLE
gdrive vs drive

### DISCUSSION THREAD

**Discussion by Triceratops9** [TheLastBen/fast-stable-diffusion#1562] [General]
I'm a bit confused about file structure in colab. Everything in colab uses "gdrive." That seems to be a different drive than "drive", where I store all my google drive documents. 

I can't figure out how to access gdrive when i'm not in colab-- 

What am I missing?

Thank you!

**Comment by TheLastBen** [comment]
when you run the first cell, it connects the gdrive of the current google account to the notebook

**Comment by Triceratops9** [comment]
Thank you!
I'm confused because sometimes there are two folders in the colab hirearchy -- Drive, which is the google drive I can easily access and then GDrive where colab puts my sessions and cpkt files. THEN sometimes colab connect where Gdrive is my google drive. The challenge is that when I've trained a model and the file structure was the first one, I can't find the sessions and models if I restart colab. What am I doing wrong?

**Reply by TheLastBen** [reply]
the default folder is `/content/gdrive/MyDrive/Fast-Dreambooth`

**Reply by remybonnav** [reply]
If you don't run the first cell to connect to you google drive, the colab will create new folder with same structure as if it was your google drive. But this is created on the google colab kernel to which you are connected. So whatever you store inside will be lost when you disconnect google colab!

**Comment by CarbonMatrix** [comment]
Replace the first code with:
```
from google.colab import drive
drive.mount('/content/gdrive/', force_remount=True)
```
This forces it to use your gdrive and not create a different gdrive. The sd folder will be created in your permanent Google Drive storage where you can put your models and VAE and they wont' get deleted.

**Comment by Rahul-i7** [comment]
I think I know what you mean, just rewrite 'gdrive' to 'drive' everywhere in the code. Make sure to change it in the paths.py as well. Worked for me...
